src/AndroidDebugLauncher/Launcher.cs

```diff
@@ -341,7 +356,7 @@ private LocalLaunchOptions SetupForDebuggingWorker(CancellationToken token)
                     {
                         fileSystem.Download(@"/system/bin/" + app_process, exePath, true);
                     }
-                    catch (AdbException) // 64-bit TODO: add 'when (is32BitTarget)'
+                    catch (AdbException)
```

Comment:
The behavior would be that if app_process_suffix is 64 and we fail to download, the AdbException would propogate, correct? And if app_process_suffix is 32, we set retry, and then if we fail to download "app_process", that AdbException will propogate? I will add this then.